package.json

```diff
@@ -48,6 +48,7 @@
   "main": "index.js",
   "repository": "https://github.com/comicrelief/serverless-starter-app",
   "dependencies": {
+    "@comicrelief/eslint-config": "^1.0.1",
```

Comment:
Shouldn't this be under `devDependencies`?

Reply:
You are right. Thanks!